docs/source/azure/examples/nsg.rst

```diff
@@ -11,7 +11,7 @@ For more examples see :ref:`azure_nsg`
          resource: azure.networksecuritygroup
          filters:
           - type: ingress
-            OnlyPorts: [22,23,24]
+            exceptPorts: [22,23,24]
```

Comment:
looks like these should be all caps with underscores per schema, same with the examples in docs/source/azure/policy/resources/nsg.rst